project/scripts/genDocsScalaLang

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
```

Comment:
Please document what this script does, and what it expects as input parameters.

Reply:
Will do